api/android/api/src/main/jni/Android-tensorflow-lite.mk

```diff
@@ -47,10 +47,13 @@ include $(PREBUILT_STATIC_LIBRARY)
 #------------------------------------------------------
 include $(CLEAR_VARS)
 
+# define types in tensorflow-lite sub-plugin
+TF_LITE_FLAGS := -DTFLITE_INT8=1 -DTFLITE_INT16=1
```

Comment:
This would be better defined where TF-LITE enable/disable is defined because TF-LITE version will be "known" when someone enable/disable TF-LITE build. That'd be around LINE 40-41 of ```/api/android/api/src/main/jni/Android.mk```

Reply:
Thanks, I moved this to android.mk.